Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -1,7 +1,7 @@
 {
   "align-items": {
     "inherited": false,
-    "initial": "normal",
+    "initial": "stretch",
```

Comment:
Wait a minute...

https://drafts.csswg.org/css-align/#align-items-property says the initial value is `normal`
https://drafts.csswg.org/css-flexbox/#align-items-property says the initial value is `stretch`

Reply:
Derp, I meant to note that in the commit message but forgot what the issue was by the time it came to write it. :facepalm:
The W3C published css-align spec is newer than the css-flexbox one, and it expands the definition of `align-items`. But since we don't support those and it's still a WD so fairly volatile, I went with the css-flexbox definition.

I'll add that to the commit message now.